### PR TITLE
Fix eslint warning about array key in Abilities.jsx

### PR DESCRIPTION
**Pull request by hooahest**
Eslint gives the following warning:

\workspace\Github\web\src\components\Hero\Abilities.jsx
  27:21  warning  Do not use Array index in keys  react/no-array-index-key

Changed 'abilities' to contain 'id' instead of relying on array index, as advised by https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

### CODE REVIEW

File: src/components/Hero/Abilities.jsx [hunk 1]
```diff
@@ -23,16 +23,16 @@ const AbilityItem = styled.div`
   padding-right: 4px;
 `;
 
-const renderAbilities = abilities => abilities.map((ability, key) => (
-  <AbilityItem key={key}>
-    <Ability {...ability} abilityID={key} />
+const renderAbilities = abilities => abilities.map(ability => (
+  <AbilityItem key={ability.key}>
+    <Ability {...ability.data} abilityID={ability.key} />
```
Comment (howardchung) [inline src/components/Hero/Abilities.jsx]: Doesn't this change the abilityID prop being passed?

Comment (howardchung) [reply src/components/Hero/Abilities.jsx]: Oh, I see we're adding the key property.

**Comment by howardchung** [comment]
I think this is still using an array index as an ID though, we're just passing it from a layer up to avoid the warning.

**Comment by bdbch** [comment]
@howardchung can we receive the dota2 internal spell id for the skill which we could for iteration keys?

**Comment by howardchung** [comment]
Yeah, presumably `key` is the key of the object in https://github.com/odota/dotaconstants/blob/master/build/abilities.json

**Pull request merged**